view-user/src/components/common/BingoResult/BingoResult.module.css

```diff
@@ -0,0 +1,53 @@
+.container {
+    /* height: 100vh; */
+    width: 100vw;
+    background-color: #C9AAAA;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .list_container {
+    height: 90vh;
+    width: 90vw;
+  }
+  
+  .card_frame {
+    border-radius: 3px;
+    border-width: 0px;
+    border-style: solid;
+    border-color: #333;
+    padding: 2rem;
+    background-color: #D9D9D9;
+  }
+
+  .card_text {
+    font-size: 3rem;
+    font-weight: bold;
+    color: #333;
+    padding-bottom: 10px;
+    padding-left: 10px;
+  }
+  
+  .card {
+    display: grid;
+    grid-template-columns: repeat(8, 1fr);
```

Comment:
```suggestion
    grid-template-columns: repeat(auto-fill, minmax(100px, 1fr));
```
auto-fillってのが幅があったらグリッドを入れるみたいだった。下記参考文献
https://ascii.jp/elem/000/001/659/1659899/

Reply:
<img width="960" alt="image" src="https://github.com/NUTFes/nutfes-Bingo/assets/106811268/6160e76f-25fa-4f8e-874a-1c22b03bc903">
一応写真はっとく